README.md

```diff
@@ -5,641 +5,291 @@
 
-In the following example we use the data masking module to anonymize persons and phone numbers in the input.
+Build your application using Maven and deploy it to Cloud Foundry:
 
-In this case, the input will be masked before the call to the LLM. However, data can not be unmasked in the LLM output.
+   ```shell
+   mvn clean package
+   cf push
+   ```
 
-```java
-final var inputParams = Map.of("privateInfo", "Patrick Morgan +49 (970) 333-3833");
-final var template =
-    ChatMessage.create().role("user").content("What is the nationality of {{?privateInfo}}");
-final var templatingConfig = TemplatingModuleConfig.create().template(template);
-
-final var maskingProvider =
-    MaskingProviderConfig.create()
-        .type(MaskingProviderConfig.TypeEnum.SAP_DATA_PRIVACY_INTEGRATION)
-        .method(MaskingProviderConfig.MethodEnum.ANONYMIZATION)
-        .entities(
-            DPIEntityConfig.create().type(DPIEntities.PHONE),
-            DPIEntityConfig.create().type(DPIEntities.PERSON));
-final var maskingConfig = MaskingModuleConfig.create().maskingProviders(maskingProvider);
-
-final CompletionPostRequest config =
-    CompletionPostRequest.create()
-        .orchestrationConfig(
-            OrchestrationConfig.create()
-                .moduleConfigurations(
-                    ModuleConfigs.create()
-                        .llmModuleConfig(LLM_CONFIG)
-                        .templatingModuleConfig(templatingConfig)
-                        .maskingModuleConfig(maskingConfig)))
-        .inputParams(inputParams);
-
-final CompletionPostResponse result =
-    new OrchestrationCompletionApi(getOrchestrationClient("default"))
-        .orchestrationV1EndpointsCreate(config);
-
-final String messageResult =
-    result.getOrchestrationResult().getChoices().get(0).getMessage().getContent();
-```
+That's it! Your application should now be running on Cloud Foundry, and the AI Core credentials are provided securely
+via service binding.
 
-See [an example in our Spring Boot application](sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java)
+---
 
-### Set model parameters
+## Documentation
 
-Change your LLM module configuration to add model parameters:
+For more detailed information and advanced usage, please refer to the following:
 
-```java
-var llmConfig =
-    LLMModuleConfig.create()
-        .modelName("gpt-35-turbo")
-        .modelParams(
-            Map.of(
-                "max_tokens", 50,
-                "temperature", 0.1,
-                "frequency_penalty", 0,
-                "presence_penalty", 0));
-```
+- [OpenAI Chat Completion](docs/guides/OPENAI_CHAT_COMPLETION.md)
+- [Orchestration Chat Completion](docs/guides/ORCHESTRATION_CHAT_COMPLETION.md)
+- [AI Core Deployment](docs/guides/AI_CORE_DEPLOYMENT.md)
 
-See [an example in our unit test](orchestration/src/test/java/com/sap/ai/sdk/orchestration/client/OrchestrationUnitTest.java)
+## Code Snippets
```

Comment:
(Minor/Comment)

I would maybe rephrase this section as FAQ for now. (Even when the questions haven't been asked frequently.)

Reply:
How about 'Troubleshooting'?